packages/pdf-renderer/README.md

```diff
@@ -1,7 +1,79 @@
 # pdf-renderer
 
-This library was generated with [Nx](https://nx.dev).
+pdf-renderer is a NestJS module used for creating pdf files out of react components.
 
-## Running unit tests
+## Installation
 
-Run `nx test pdf-renderer` to execute the unit tests via [Jest](https://jestjs.io).
+```
+npm install @leancodepl/pdf-renderer
+yarn add @leancodepl/pdf-renderer
+```
+
+## Prerequisites
+
+For the pdf-renderer to run correctly, it is required to use Dockerfile delivered with a package, which installs all of
+the dependencies needed by Puppeteer. <br/>
+
+If you want to provide custom fonts, you need those in the `.woff` format, which is the only one supported at the
+moment. <br/>
+
+Keep in mind, that everything you pass into the renderer, has to be synchronous, for it to work as expected. Any data
+component needs, should be provided beforehand. Also, component shouldn't call any hooks.
```

Comment:
This isn't exactly true - component can call hooks, but since the synchronicity requirement it might not work as expected (for example `useEffect`). `useMemo` or `useCallback` doesn't make any sense but will work. However you can still use some custom hooks to extract some parts of logic for components.